app/models/institution_builder.rb

```diff
@@ -558,4 +559,34 @@ def self.build_institution_programs(version_number)
     sql = InstitutionProgram.send(:sanitize_sql, [str, version_number])
     InstitutionProgram.connection.execute(sql)
   end
+
+  def self.build_versioned_school_certifying_official(version_number)
+    str = <<-SQL
+      INSERT INTO versioned_school_certifying_officials(facility_code,
```

Comment:
Looking better.  Can you move `facility_code` to the line below?

Reply:
Missed that one, I went ahead and updated it.